sdk/containerregistry/azure-containers-containerregistry/src/main/java/module-info.java

```diff
@@ -7,12 +7,9 @@
     exports com.azure.containers.containerregistry;
     exports com.azure.containers.containerregistry.models;
 
-    exports com.azure.containers.containerregistry.implementation to com.azure.core;
-    exports com.azure.containers.containerregistry.implementation.models to com.azure.core;
-
-    // exporting some packages specifically for Jackson
-    opens com.azure.containers.containerregistry to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.containers.containerregistry.implementation to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.containers.containerregistry.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.containers.containerregistry.models to com.fasterxml.jackson.databind, com.azure.core;
+
+    // This is still needed even though we export the models.
```

Comment:
Correct, `exports` offers the package's public API as accessible to other modules. `opens` offers the package as available through reflection.